### DISCUSSION TITLE
Provided an easier way for users to find organizations

### DISCUSSION THREAD

**Discussion by michael-tharrington** [forem/forem#16300] [Feature Requests]
**Is your feature request related to a problem? Please describe.**

Folks occasionally want to browse/search and see what orgs exist on DEV and there's no easy way to get that information.

**Describe the solution you'd like**

1. I think it'd be good if we added `organizations` as a category/filter to search for.
2. I think it'd be helpful if we added https://dev.to/organizations as a page that listed out the many orgs that are created on DEV. I think we'd want to try and filter out the spam or super small abandoned orgs. We might want to treat it a bit like https://dev.to/tags and show only the top orgs by pageviews or members joined. Maybe orgs should have to get verified to appear on the page.

**Additional context**

This request is related to these 2 previous requests:
- https://github.com/forem/forem/issues/9775
- https://github.com/forem/internalEngineering/issues/368

I've heard this request from various personas:
- Creators (Web Monetization definitely, but other Creators who plan to use organizations as a key feature will also likely want a way to showcase organizations on their Forem)
- Users on DEV have asked where they can see all orgs so that they can choose if they want to follow them.
- Org admin also would like to get more visibility on their org/make it easier for folks to discover them

**Comment by FidalMathew** [comment]
@michael-tharrington is this issue currently in progress?

**Comment by TERNOX** [comment]
I believe that a list of organizations is a must have, ideally sorted by the date of the last post in the organization.
